_dev/apps/ui/src/store/modules/campaigns/state.ts

```diff
@@ -85,6 +85,7 @@ export type CampaignObject = {
   productFilters?: ProductsFilteredObject[];
   status?: CampaignStatus|CampaignStatusToggle;
   hasUnhandledFilters?: boolean;
+  isBudgetBelowMinimum?: boolean;
```

Comment:
@tberger-prestashop This variable has to be updated with the additional data from the API